types/11ty__eleventy-plugin-directory-output/index.d.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference types="node" />
```

Comment:
Does this actually depend on node?

Reply:
Not directly, but `11ty.ts` uses them without importing it or specifying it as a dependency.

Reply:
Sure, though I would almost think that it could just be a dev dep here since the types are unreferenced (and users are expected to have installed it anyway). Just feels awkward for unrelated packages to be pulling in node types. But, it's okay.

Reply:
Apparently it’s not needed. I must have confused it with something else.